reward_extraction/behavior_cloning/behavior_cloning.py

```diff
@@ -0,0 +1,191 @@
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.functional as F 
+from torch.utils.data import Dataset, DataLoader
+from typing import Tuple
+from robot.data import RoboDemoDset
+from robot.base_robot import XArmTaskSpaceEnv
+# export PYTHONPATH="${PYTHONPATH}:/home/xarm/Documents/JacobAndDavin/test/rewardlearning-robot"
+device = 'cuda' if torch.cuda.is_available() else 'cpu'
+class BCNet(nn.Module):
+    def __init__(self) -> None:
+        super(BCNet, self).__init__()
+        self.fc1 = nn.Linear(3, 30)
```

Comment:
You can just use this instead of writing a MLP every time

https://github.com/dxyang/rewardlearning-robot/blob/main/reward_extraction/models.py#L20

```
from reward_extraction.models import MLP

self.net = MLP(3, 300, 3, 3, nn.Tanh())
```

Reply:
oh that's nice, I will look into that module. Just update BCNet to MLP.